Library/Formula/embulk.rb

```diff
@@ -18,7 +18,7 @@ def install
 
   test do
     system bin/"embulk", "example", "./try1"
-    system bin/"embulk", "guess", "./try1/example.yml", "-o", "config.yml"
+    system bin/"embulk", "guess", "./try1/seed.yml", "-o", "config.yml"
```

Comment:
This change reflects this upstream changes: https://github.com/embulk/embulk/pull/377.